examples/03_connectivity/plot_compare_resting_state_decomposition.py

```diff
@@ -9,13 +9,9 @@
 spatial maps. It extracts maps that are naturally sparse and usually cleaner
 than ICA
 
-    * Gael Varoquaux et al.
-      Multi-subject dictionary learning to segment an atlas of brain spontaneous
-      activity
-      Information Processing in Medical Imaging, 2011, pp. 562-573, Lecture Notes
-      in Computer Science
-
-Available on https://hal.inria.fr/inria-00588898/en/
+   * Arthur Mensch et al. `Compressed online dictionary learning for fast resting-state fMRI decomposition
+   <https://hal.archives-ouvertes.fr/hal-01271033/>`_,
```

Comment:
Probably, you may fix CircleCI doc error by indenting the line like below: Second line has indentation error.

* Arthur Mensch et al. `Compressed online dictionary learning for fast
  resting-state fMRI decomposition 
  <https://hal.archives-ouvertes.fr/hal-01271033/>`_,
  IPMI 2016, Lecture Notes in Computer Science